wordpress-mobile/WordPress-iOS
# Update Create icon

This PR updates the Create / Write icon used in the tab bar, as per Automattic/gridicons#256.

![tab-write](https://user-images.githubusercontent.com/4780/32721947-2c808dd6-c860-11e7-8bce-c6c7c9bfc55c.png)

To test:

* Build and run the app.
* Check the appearance of the Write button in the tab bar is correct in both portrait and landscape.
